src/pages/edit-feedback-page/components/edit-feedback-form/EditFeedbackForm.tsx

```diff
@@ -0,0 +1,192 @@
+import { Formik } from 'formik'
+import { useEffect, useState } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import { fetchDataByFeedbackId } from '../../..'
+import {
+  deleteFeedback,
+  Feedback,
+  updateFeedback,
+} from '../../../../services/feedback.service'
+import { Dropdown } from '../../../../components/Dropdown'
+import {
+  categoryMap,
+  optionsCategory,
+  optionsStatus,
+  statusMap,
+} from './constants'
+import { editFeedbackSchema } from './validation'
+import { ClipLoader } from 'react-spinners'
+
+type FormValues = {
+  id: number
+  detail: string
+  title: string
+  status: string
+  category: string
+}
+
+export function EditFeedback() {
+  const [feedback, setFeedback] = useState<Feedback | undefined>()
+  const [isLoading, setIsLoading] = useState(true)
+
+  const { id } = useParams()
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    fetchDataByFeedbackId(id).then(async (data) => {
+      setFeedback(data[0])
+      setIsLoading(false)
+    })
+  }, [])
+
+  const handleDeleteFeedback = async () => {
+    await deleteFeedback(Number(id))
+    navigate('/')
+  }
+
+  const handleUpdateFeedback = async ({
+    id,
+    detail,
+    title,
+    status,
+    category,
+  }: FormValues) => {
+    await updateFeedback(Number(id), detail, title, status, category)
+    navigate('/')
```

Comment:
Here we could also return back to feedbackPage, don't know if it's better but as an option to consider.